utilities/pipelines/staticValidation/compliance/module.tests.ps1

```diff
@@ -83,9 +83,13 @@ Describe 'File/folder tests' -Tag 'Modules' {
                     moduleType                         = $moduleType
                     moduleFolderName                   = $resourceTypeIdentifier
                     moduleFolderPath                   = $moduleFolderPath
+                    moduleVersionExists                = Test-Path (Join-Path -Path $moduleFolderPath 'version.json')
                     isTopLevelModule                   = ($resourceTypeIdentifier -split '[\/|\\]').Count -eq 2
                     childModuleAllowedList             = $childModuleAllowedList
                     childModuleAllowedListRelativePath = $childModuleAllowedListRelativePath
+                    # TODO remove after pilot phase
```

Comment:
@ReneHezser are you sure you merged latest? From this diff I feel that is not done